prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -261,8 +261,8 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 			   ntohs(av_entry->addr.sin_port), av_entry);
 
 		av->used++;
-		ret++;
 	}
+	ret = count - failed;
```

Comment:
`failed` is incremented in error case only when `(av->flags & FI_EVENT)` is true so `count - failed` may be more than the actual # of addresses inserted.

Reply:
Good point; I'm not hugely familiar with the verbs provider codebase and must have missed that. I've pushed another commit that always increments `failed` on error. That way, we count insertions and failures the same way, whether `FI_EVENT` is set or not.